### ISSUE TITLE
dnsdist: unclear wording in remote logger stats

### ISSUE THREAD

**Issue by yantarou** [PowerDNS/pdns#10405]
- Program: dnsdist
 - Issue type: Bug report (well, a cosmetical one)

### Short description

When using a remote logger...

```
rl = newRemoteLogger("127.0.0.1:12345", 10, 100, 15)
addAction(AllRule(), RemoteLogAction(rl))
addResponseAction(AllRule(), RemoteLogResponseAction(rl))
```
...and afterwards checking the stats in the console...

```
> showRules()
#     Matches Rule                                                     Action
0    78002097 All                                                      remote log to 127.0.0.1:12345 (135898947 queued, 0 dropped)
```

...it seems like all packets are still being queued (i.e. not sent, but also not dropped).

According to @habbie this may actually just be some wrong wording, i.e. `queued` actually meaning `sent`.

### Environment
 - Software version: 1.5.2
 - Software source: PowerDNS repository

### Steps to reproduce

1. Configure remote logging
2. Check the stats

### Expected behaviour

Clear wording that does not confuse the user.

The `queued` stat should show the currently queued packets.

I don't think it really makes sense to just rename it to `sent`, because the user likely doesn't care about how many million packets have been sent. (Also, will we hit some overflow there on high-utilized systems?)

The important values are `queued` to see if the queue size should be adjusted, and the amount of `dropped` packets.

**Comment by rgacogne** [comment]
The wording is correct, `queued` means "placed into the in-memory circular buffer storing the data that will be sent to the remote endpoint as soon as possible" (during the next `flush` event). For performance reasons we send the data as a stream, so from that point on we don't know anymore how many messages are actually sent, only bytes, meaning we have no way to provide you with the metric you are looking for, sorry. What we could do, however, is record how many bytes were queued and how many were actually sent, but that's the best we can do.
I don't understand why you would like to see the number of queued items in your case, though? If you really want to see if the queue size should be adjusted, then `dropped` is what you want.

**Comment by yantarou** [comment]
Hi @rgacogne,

I can think of different meanings of "queued":

1) Put into the queue over dnsdist's "lifetime" (i.e. steadily increasing, no matter if sent or dropped)
2) Currently in the queue

My point is that 1) doesn't have any benefit to the user, and may just create confusion.

> I don't understand why you would like to see the number of queued items in your case, though? If you really want to see if the queue size should be adjusted, then dropped is what you want.

Exactly my point.

Assuming that 1) is true, `queued` doesn't give any benefit to the user, but may make them assume that packets still are in the queue without being sent. (That's how I interpreted it when I opened this ticket.)
